mr/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerWithEngine.java

```diff
@@ -150,6 +150,27 @@ public void testScanTable() throws IOException {
     Assert.assertArrayEquals(new Object[] {"Alice", 0L}, descRows.get(2));
   }
 
+  @Test
```

Comment:
I think these queries do not use the execution engine itself, so we do not have to put these tests to this class.
It is ok to have them only in the TestHiveIcebergStorageHandlerLocalScan.java

Reply:
OK, i'll remove the test code in TestHiveIcebergStorageHandlerLocalScan.java